src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/JpaTimelinePersistenceAdapter.kt

```diff
@@ -22,9 +19,6 @@ class JpaTimelinePersistenceAdapter(
   private val timelineRepository: TimelineRepository,
   private val timelineMapper: TimelineEntityMapper,
   private val timelineEventMapper: TimelineEventEntityMapper,
-  // TODO RR-566 - call prisonApiClient via service class
-  private val prisonApiClient: PrisonApiClient,
-  @Value("\${featureToggles.call-prison-api-enabled}") private val callPrisonApiEnabled: Boolean,
 ) : TimelinePersistenceAdapter {
```

Comment:
This class has simply been reverted back to its previous state, before I tested the connection with the prison-api.